glue_solar/core/core.py

```diff
@@ -11,6 +11,7 @@
 from glue.core import Component, Data
 from glue.core.visual import VisualAttributes
 from glue.core.data_factories import is_fits
+from glue.core.coordinates import WCSCoordinates
```

Comment:
Is this import used anywhere?

Reply:
Not anymore, and so I have removed the unused import statements.